models/evc.py

```diff
@@ -112,9 +112,7 @@ def __init__(self, controller, **kwargs):
         self.secondary_constraints = kwargs.get("secondary_constraints", {})
         self.creation_time = get_time(kwargs.get("creation_time")) or now()
         self.owner = kwargs.get("owner", None)
-        self.sb_priority = kwargs.get("sb_priority", None) or kwargs.get(
-            "priority", None
-        )
+        self.sb_priority = kwargs.get("sb_priority", self.get_priority())
```

Comment:
It is okay to insert the default values here? Inside the class `EVCBase`. Or maybe inside `Main._evc_dict_with_instances()`, after [line 803](https://github.com/kytos-ng/mef_eline/blob/2c92011d90bdad41bdfe230b0870489704f9fba7/main.py#L803) have a flag:
```
                    if value["tag"]:
                        uni_flag = True
```
Before returning data, this:
```
        if uni_flag:
            if not data["sb_priority"]:
                data["sb_priority"] = settings.EVPL_SB_PRIORITY
        else:
            if not data["sb_priority"]:
                data["sb_priority"] = settings.EPL_SB_PRIORITY
```

Reply:
@Alopalao, this current approach, looks good and very easy to evolve and maintain. It cover the base case and when both ends of an EVC are symmetric it works great. However, if we have tag/no-tag ends then I think this would incorrectly set `EVPL_SB_PRIORITY` for the untagged end too. 

I think we could augment [_prepare_uni_flows](https://github.com/kytos-ng/mef_eline/blob/master/models/evc.py#L829), to later on try to overwrite the push flow of the no-tag end to use the default priority. It might be something to explore. You can also double check with @italovalcy to see if he's got any other suggestion about this.